api/views.py

```diff
@@ -121,3 +143,174 @@ def get(self, request):
             return Response({"isAuthenticated": False}, status=status.HTTP_403_FORBIDDEN)
 
         return Response({"username": request.user.username, "id": request.user.id, "isAuthenticated": True})
+
+
+class CartItemSerializer(serializers.Serializer):
+    product_id = serializers.IntegerField()
+    quantity = serializers.IntegerField()
+
+
+class CartItem(NamedTuple):
+    product_id: int
+    quantity: int
+
+
+class CartSerializer(serializers.Serializer):
+    cart = CartItemSerializer(many=True)
+    hash = serializers.CharField()
+    subtotal_cents = serializers.IntegerField()
+    total_cents_with_fee = serializers.IntegerField()
```

Comment:
All of the cart stuff can go. that was for the react store which no longer exists